src/assets/img/triangle-left.svg

```diff
@@ -0,0 +1 @@
+<?xml version="1.0" standalone="no"?><!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd"><svg class="icon" width="200px" height="200.00px" viewBox="0 0 1024 1024" version="1.1" xmlns="http://www.w3.org/2000/svg"><path d="M320 511.872l383.744 383.744V128.128L320 511.872z" fill="#666666" /></svg>
```

Comment:
Is this file and triangle-right.svg used anywhere in this commit? If no, these .svg files should not be included in this commit.